robotfm_tests/docs/integration_packs_doc.rst

```diff
@@ -12,7 +12,7 @@
 
 
     Verify multiple packs can be installed from repo
-        ${result}=          Run Process  st2  run  packs.install  packs\=${PACK TO INSTALL 1},${PACK TO INSTALL 2}  repo_url\=${BASE REPO URL}/${INSTALL FROM REPO}  -j
```

Comment:
I thought we are going to keep alias for backward compatibility or similar. Or am I missing something?